lib/api/objectDelete.js

```diff
@@ -169,8 +170,14 @@ function objectDelete(authInfo, request, log, cb) {
                     result.versionId : versionIdUtils.encode(
                         result.versionId, config.versionIdEncodingType);
             }
+            const versioningSuspended = bucketMD
+                && bucketMD._versioningConfiguration
+                && bucketMD._versioningConfiguration.status === 'Suspended';
+            const hasByteLength = objectMD && objectMD['content-length'];
+            const byteLength = versioningSuspended && hasByteLength ? objectMD['content-length'] : null;
             pushMetric('putDeleteMarkerObject', log, {
                 authInfo,
+                byteLength,
```

Comment:
Is `byteLength` necessarily from the null version going to be deleted? It does not seem so to me unless I'm mistaken, see what `preprocessingVersioningDelete` is doing to derive info about the null version to delete. I think it may not be what is in `objectMd`.